### PR TITLE
unzip 64-bit only + CVE-2014-9913 & CVE-2018-1000035

### PR DESCRIPTION
**Pull request by Mno-hime**
Runtested.

The trick to make it 64-bit in `COMPONENT_BUILD_TARGETS` is really cheap, but the only way I was able to persuade the build system to build in 64-bits.

### CODE REVIEW

File: components/archiver/unzip/Makefile [hunk 1]
```diff
@@ -48,25 +48,26 @@ COMPONENT_BUILD_ENV += \
  "LOCAL_UNZIP=-DUNICODE_SUPPORT -DNO_WORKING_ISPRINT -DUNICODE_WCHAR -DNO_LCHMOD"
 
 # The Makefile we want is under unix/Makefile
-COMPONENT_BUILD_ARGS += -f unix/Makefile
-COMPONENT_BUILD_TARGETS += generic CC="$(CC)"
-COMPONENT_INSTALL_ARGS += -f unix/Makefile
+COMPONENT_BUILD_ARGS    += -f unix/Makefile
+COMPONENT_BUILD_TARGETS += generic CC="$(CC) -m64"
```
Comment (alarcher) [inline components/archiver/unzip/Makefile]: Could use CC_BITS here.

File: components/archiver/unzip/Makefile [hunk 2]
```diff
@@ -48,25 +48,26 @@ COMPONENT_BUILD_ENV += \
...
 
-install:	$(INSTALL_32)
+install:	$(INSTALL_64)
 
-test:		$(NO_TESTS)
+test:		$(TEST_64)
 
 BUILD_PKG_DEPENDENCIES =	$(BUILD_TOOLS)
 
-include $(WS_TOP)/make-rules/depend.mk
```
Comment (alarcher) [inline components/archiver/unzip/Makefile:69]: This line can be removed.

File: components/archiver/unzip/unzip.p5m [hunk 3]
```diff
@@ -1,28 +1,16 @@
...
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
 #
 
-<transform file path=usr.*/man/.+ -> default mangler.man.stability volatile>
-
 set name=pkg.fmri \
     value=pkg:/compress/unzip@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
```
Comment (alarcher) [inline components/archiver/unzip/unzip.p5m]: Could move the package metadata to the Makefile.

File: components/archiver/unzip/Makefile (hunk 2 above)
Comment (Mno-hime) [reply components/archiver/unzip/Makefile:69]: Removed the include.

**Pull request merged**